apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerGrid.tsx

```diff
@@ -53,20 +53,19 @@ type NftSelectorPickerGridProps = {
     sortView: NftSelectorSortView;
   };
   screen: ScreenWithNftSelector;
-  queryRef: NftSelectorPickerGridFragment$key;
+  // queryRef: NftSelectorPickerGridFragment$key;
```

Comment:
nit: remove the comment

Reply:
oops good catch , had auto merge on lol ill fix in follow up PR